extensions/registries/src/extension.ts

```diff
@@ -19,7 +19,7 @@ import * as extensionApi from '@podman-desktop/api';
 
 import dockerIoImage from './images/docker.io.png';
 import gcrIoImage from './images/gcr.io.png';
-import ghcrIoImage from './images/ghcr.io.png';
+import ghcrIoImage from './images/ghcr.io.svg';
```

Comment:
might need to create a Svelte component with the svg content in it.

Example https://github.com/podman-desktop/podman-desktop/blob/516922e2ef043c208d55e9921edda79f7798aa20/packages/ui/src/lib/icons/ContainerIcon.svelte